test/unit/test_aiakos.yml

```diff
@@ -0,0 +1,12 @@
+#----------------------------------------------------------------------------
+# Aiakos application section (mandatory)
+#----------------------------------------------------------------------------
+app:
+  port: 3000		# App listen port
+
+#-----------------------------------------------------------------------------
+#  openstack admin (mandatory)
+#-----------------------------------------------------------------------------
+openstack:
+  username: ''		# admin openstack user
+  password: ''		# admin openstack password
```

Comment:
You cannot use fake values in the username and password to check them in the unit tests?